src/plugins/intel_gpu/tests/fusions/gemm_fusion_test.cpp

```diff
@@ -376,19 +379,26 @@ TEST_P(gemm_2in_act_scale_eltwise, broadcast_eltwise) {
         reorder("reorder_bfyx", input_info("sum"), p.default_format, data_types::f32)
     );
     // Activation won't be fused because onednn doesn't support negative activation
-    if (engine.get_device_info().supports_immad && !p.kernel_name.empty())
+    if (engine.get_device_info().supports_immad) {
         p.expected_fused_primitives += 2;
+        if (!p.kernel_name.empty())
+            p.expected_not_fused_primitives += 2;
+    }
```

Comment:
we can support `negative` activation by mapping it into `eltwise_linear`

Reply:
Done.